pkgs/by-name/le/less-css/plugins/clean-css/update.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq common-updater-scripts nodejs prefetch-npm-deps
+
+set -xeu -o pipefail
+
+PACKAGE_DIR="$(realpath "$(dirname "$0")")"
+cd "$PACKAGE_DIR/.."
+while ! test -f flake.nix; do cd .. ; done
+NIXPKGS_DIR="$PWD"
+
+latest_commit="$(
+  curl https://api.github.com/repos/less/less-plugin-clean-css/branches/master \
```

Comment:
```suggestion
  curl -L -s ${GITHUB_TOKEN:+-u ":${GITHUB_TOKEN}"} https://api.github.com/repos/less/less-plugin-clean-css/branches/master \
```
otherwise you easily run into rate limits.